src/forms/BoundSelectField.test.tsx

```diff
@@ -62,6 +62,30 @@ describe("BoundSelectField", () => {
     // Then formState has the latest value when onBlur is called
     expect(autoSave).toBeCalledWith("s:2");
   });
+
+  it("shows dirty indicator", async () => {
+    const author = createObjectState(formConfig, { favoriteSport: "s:1" });
+    const r = await render(<BoundSelectField field={author.favoriteSport} options={sports} isDirtyIndicator />);
+    r.favoriteSport().focus();
+    click(r.getByRole("option", { name: "Soccer" }));
+    expect(r.favoriteSport_isDirtyIndicator()).toBeInTheDocument();
+  });
+
+  it("ignores isDirtyIndicator if fieldDecoration is provided", async () => {
+    const author = createObjectState(formConfig, { favoriteSport: "s:1" });
+    const r = await render(
+      <BoundSelectField
+        field={author.favoriteSport}
+        options={sports}
+        isDirtyIndicator
+        fieldDecoration={() => <div data-testid="lookforme" />}
+      />,
+    );
+    r.favoriteSport().focus();
+    click(r.getByRole("option", { name: "Soccer" }));
```

Comment:
I could not get `select(r.favoriteSport, "Soccer")` to work and idk why. Gave some "length of 0" error.

Reply:
`select` helper expects the element to be a `<select>`, which is not the case here.